tests/model/SiteTreeTest.php

```diff
@@ -823,6 +859,14 @@ public function testIsSection()
         $this->assertTrue($ceo->isSection());
     }
 
+    public function testURLSegmentReserved()
```

Comment:
Although this is a double-up from previous tests, I believe it is a good test to have, to isolate `admin` testing and have it explicitly tested